assets/src/edit-story/components/canvas/carousel/leftArrow.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 8 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
In @miina's PR for page navigation, [she created similar simple buttons using CSS](https://github.com/ampproject/amp-wp/pull/4049/files#diff-850b4bbccd4c90c7be2925a731b58e5eR40-R67). We should probably use the same strategy in the two PR's, whichever is deemed better.